seigniorage.tex

```diff
@@ -1,230 +1,91 @@
-
-We set $q_{OTF}$ as a protocol parameter and define
-
+Instead of forcing the protocol to keep track of changes to supply on a block-by-block basis, we take the final supply of the previous era as a benchmark, to calculate how much the round reward should be for a given era, throughout which it stays constant. This greatly reduces the complexity of production code without sacrificing anything crucial to seigniorage. Thus, we use the following recursive definition of CLX supply for seigniorage:
+%
 \begin{equation}
-I = \{i\in \mathbb{N} \mid \boldsymbol{w}(V_i)\geq q_{OTF}\}
+  \begin{aligned}
+    Q_0 &= Q_0 \\
+    Q_{n+1} &= Q_n
+    + \texttt{rewards}_n
+    - \texttt{slashings}_n
+  \end{aligned}
 \end{equation}
+%
+where $Q_n$ is the supply at the beginning of era $n$, $\texttt{rewards}_n$ is the total CLX minted per seigniorage during era $n$, and $\texttt{slashings}_n$ is the total CLX slashed during era $n$. All of these quantities can be derived objectively from the history of the global state.
 
-to be the set of ticks where the assigned weight is sufficient for OTF. Such ticks are said to be \emph{feasible}, and the ticks that do not satisfy this condition are said to be \emph{infeasible}. Given a feasible tick, the corresponding the rounds that start on it are also said to be feasible. Only feasible rounds are eligible for rewards, and participating in an infeasible round does not earn any reward. In other words, a round's finality can be rewarded only if enough validators have announced to vote on it.
-
-It is ideal to have the validators converge on the same round exponent, because then the participated weight would be maximized for each round. For that reason, reward allocated per round is a function of the total weight assigned to the corresponding tick.
-
-\subsection{Finality Criteria}
-\label{sec:finality-criteria}
-
-\subsubsection*{On-Time Finalization}
-\label{sec:on-time-finalization}
-
-OTF is concerned with the finality that is achieved in the short run, with in-round messages. For the sake of detecting this short-term finality, we restrict ourselves to \emph{summits that are achievable only by in-round messages}.
-
-A summit is defined as $\mathcal{S}=((C_j)_{j\in\mathbb{N}}, \sigma,p,p',q)$ where the respective properties $p$ and $p'$ are determined by the fork choice rule. We are interested in summits achievable only by valid in-round messages, so we introduce the property
-
+We have previously introduced the minimum round exponent, $n^\ast$, which all the validators should operate at, under ideal conditions. This allows us to derive the seigniorage rate for an optimal round
+%
 \begin{equation}
-  p_i = \{\mu \mid i \leq T(\mu)< i+2^{n_{S(\mu) }(i)}\}
+  \sigma^\ast = \pow(1 + \sigma, 2^{n^\ast}/\text{ticks\_per\_year}) - 1
```

Comment:
Since this is kept constant for a year, shouldn't we divide instead of taking the root?
```suggestion
  \sigma^\ast = \sigma \cdot 2^{n^\ast} / \text{ticks\_per\_year}
```

Reply:
No, the relation should be correct. We keep updating the supply, and multiply \sigma^\ast with the supply of the previous era. It would be like you said if we always multiplied it with inital supply.

(Btw \sigma^\ast is always kept constant, not just during the year. Unless it's decided to change the seigniorage rate.)